src/nvim/menu.c

```diff
@@ -127,7 +128,7 @@ ex_menu (
     }
     arg = skipwhite(arg);
   } else if (eap->addr_count && eap->line2 != 0) {
-    pri_tab[0] = eap->line2;
+    pri_tab[0] = (int) eap->line2;
```

Comment:
Hi @philix. Glad to see you again here.
We've had the policy of not allowing unchecked casts in new/modified code, so I'd wait for this to be fixed.